src/core/common/message.hpp

```diff
@@ -476,20 +477,21 @@ class Message : public Buffer
     Message *Clone(void) const { return Clone(GetLength()); };
 
     /**
-     * This method returns the datagram tag used for 6LoWPAN fragmentation.
+     * This method returns the datagram tag used for 6LoWPAN fragmentation or the identification used for IPv6
+     * fragmentation.
```

Comment:
nit: update the `@returns` below as well?

Reply:
Yes, updated.